lib/jerboa/format/body/attribute/lifetime.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Jerboa.Format.Body.Attribute.Lifetime do
+  @moduledoc """
+  LIFETIME attribute as defined in [TURN RFC](https://trac.tools.ietf.org/html/rfc5766#section-14.2)
+  """
+
+  alias Jerboa.Format.Body.Attribute
+  alias Jerboa.Format.Lifetime.LengthError
+
+  defstruct duration: 0
+
+  @max_duration :math.pow(2, 32) - 1
+
+  @typedoc """
+  Represents a lifetime of the allocation
+
+  * `:duration` is a duration of a lifetime in seconds
+  """
+  @type t :: %__MODULE__{
+    duration: non_neg_integer
+  }
+
+  @doc false
+  @spec encode(t) :: binary
+  def encode(%__MODULE__{duration: duration})
+    when is_integer(duration) and duration <= @max_duration and duration >= 0 do
```

Comment:
https://hexdocs.pm/elixir/Kernel.html#in/2

Good thing is that for ranges, `in/2` translates them to:
```elixir
a in 1..3
# ==
a >= 1 and a <= 3
```

Bad thing is that it won't check if the value is integer, because floats can also qualify the condition.